icu4c/source/configure

```diff
@@ -9128,16 +9128,16 @@ then
 else
     echo "Spawning Python to generate data/rules.mk..."
     PYTHONPATH="$srcdir/data" $PYTHON -m buildtool \
-        --format gnumake \
+        --mode gnumake \
         --seqmode parallel \
-        --glob_dir "$srcdir/data" \
+        --in_dir "$srcdir/data" \
```

Comment:
Optional: Align the --in_dir option name with the --sourcedir option name in ICU binary tools?

Reply:
Hmm.  I use the name `IN_DIR` not only here but throughout the buildtool Python code base.  The other options are named similarly, `--out_dir` and `--tmp_dir`.  I think I'll keep it named `--in_dir`.